learners/bestarmid/fixconf/correlatedlearner/utils.py

```diff
@@ -14,6 +16,10 @@ class CorrelatedLearner(FixConfBAILearner):
   def name(self):
     pass
 
+
+  def _mat_norm(self, x, A):
```

Comment:
Can you put this outside class and add `__all__ = ['FixConfBAILearner', 'mat_norm']` before class definition?

Reply:
Does this look correct [in correlatedlearner/utils.py]?

<img width="432" alt="Screen Shot 2020-02-24 at 12 02 38 PM" src="https://user-images.githubusercontent.com/5779125/75186826-96816b00-56fd-11ea-887b-f013ba1bf91f.png">

Reply:
Yes, you are right. Two lines before and after `mat_norm` definition.